modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/Subscriptions/index.js

```diff
@@ -0,0 +1,56 @@
+import {useQuery} from '@apollo/client';
+import React, {useState} from 'react';
+import {getAccountSubscriptions} from '../../../../common/services/liferay/graphql/queries';
+import CardSubscription from '../CardSubscription';
+import SubscriptionsNavbar from '../SubscriptionsNavbar';
+
+const Subscriptions = ({accountKey}) => {
+	const [selectedTag, setSelectedTag] = useState(() => '');
```

Comment:
State value should be static values, not callback or functions

```suggestion
	const [selectedTag, setSelectedTag] = useState('');
```

Reply:
Fixed, changes are on commit **4287c0c**.